app/frontend/src/components/base/BaseSecure.vue

```diff
@@ -1,21 +1,41 @@
 <template>
   <div v-if="authenticated">
-    <div v-if="authorized()">
-      <slot />
+    <div v-if="isUser">
+      <div v-if="isAdmin">
+        <slot />
+      </div>
+      <div v-else class="text-center">
+        <h1 class="my-8">You are not authorized to use this feature.</h1>
+        <router-link :to="{ name: 'About' }">
+          <v-btn color="primary" class="about-btn" large>
+            <v-icon left>home</v-icon>
+            <span>About</span>
+          </v-btn>
+        </router-link>
+      </div>
     </div>
+    <!-- TODO: Figure out better way to alert when user lacks chefs user role -->
     <div v-else class="text-center">
-      <h1 class="my-8">You are not authorized to use this feature.</h1>
-      <router-link :to="{ name: 'Home' }">
+      <h1 class="my-8">
+        Your account is not set up correctly. Please contact support.
```

Comment:
Should ask Matt/Chris what to put here, it should have some contact info.  Not good to say contact support without any contact information.

Reply:
Updated as per discussions.